etc/ctrl.yml

```diff
@@ -12,16 +12,16 @@ v: 3
 # Define all of the paths to the certificates and key required to establish the controller's identity.
 #
 identity:
-  cert:                 etc/ca/intermediate/certs/ctrl-client.cert.pem
-  server_cert:          etc/ca/intermediate/certs/ctrl-server.cert.pem
-  key:                  etc/ca/intermediate/private/ctrl.key.pem
-  ca:                   etc/ca/intermediate/certs/ca-chain.cert.pem
+  cert: ${ZITI_SOURCE}/ziti/etc/ca/intermediate/certs/ctrl-client.cert.pem
```

Comment:
I duplicated the value of `identity` from the other provided controller config `ctrl.with.edge.yml`. My goal is to express a convention across the controller configs so that it's simple to guide the reader to initialize their shell environment in a way that works for all of the sample configs. 

There are a couple of downsides to this however, and I'm wondering if we can change the convention: `${ZITI_SOURCE}/ziti/`. Downsides include:
1. This assumes `ZITI_SOURCE` is the _parent_ directory of the checked-out copy. This is a little unexpected and requires explanation. It would be more intuitive if this was the repo copy itself, not the parent dir. There doesn't seem to be any requirement to share data between repos and so it seems like we could optimize for a self-contained experience of this repo alone.
2. This assumes the directory name of this repo's checked out copy is exactly "ziti", which is not a safe assumption. Is there a broader reason today why devs must _always_ check out to the default name "ziti"? If so, then we should explain that reasoning, and if not then we should only operate on files within this repo.

Reply:
The configs in `ziti/etc` have always been meant for local development that dictates the project be checkout in a specific manner. The way they are set up is because it caused the least amount of friction with our local development when environment variable injection was introduced to the configuration files.

If you want, you can make a set of configurations that are specifically targeted at quickstart, articles, etc. I personally would like you to not change the existing dev configs.

Additionally, the ziti configuration files for the router and controller allow for any environment variable to be used in the `${...}` format. So if you wish, when you create your configuration files for your purposes with whatever environment variables you wish.

Reply:
Thanks @andrewpmartinez . It not necessary to change anything in `ziti/etc` with this PR and I've reverted the one change I had made there to the fabric-only controller config. 

My main goal with this PR is to add a few pointers and hints to guide the person who has newly cloned this repo and wishes to build from source and run a minimal, local Ziti stack they can use for development.